skit_calls/cli.py

```diff
@@ -68,15 +68,29 @@ def process_date_filters(
     """
     today = date.today()
 
+    start_minute_offset, end_minute_offset = 0, 0
+
+    if len(start_time_offset) == 4:
```

Comment:
as `start_time_offset` is type `int`, doing len will through error, please cast it as str first